jobs/rocoto/gempakmetancdc.sh

```diff
@@ -0,0 +1,17 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
+status=$?
+if (( status != 0 )); then exit "${status}"; fi
+
+export job="gempakmetancdc"
+export jobid="${job}.$$"
+
+# Execute the JJOB
+"${HOMEgfs}/jobs/JGDAS_ATMOS_GEMPAK_META_NCDC"
```

Comment:
Is this only for gdas? Is there a gfs version of the job? Can't remember what happens in ops off the top of my head.

Reply:
Also, if you didn't already, review the gempaks jobs in this ops ecflow def file: https://github.com/NOAA-EMC/global-workflow/blob/dev/gfs.v16/ecf/defs/gfs_v16_3.def

Reply:
both:
- `gempak`

gdas-only:
- `gempakmetancdc`

gfs-only:
- `gempakmeta`
- `gempakncdcupapgif`
- `gempakpgrb2spec`

Reply:
Done.